lib/cell.rb

```diff
@@ -27,10 +27,10 @@ def fire_upon
   end
 
   def render(reveal = false)
-    return 'S' if reveal = true && !empty? && !fired_upon?
-    return '.' if fired_upon? == false
     return "M" if fired_upon? == true && empty?
-    return 'H' if fired_upon? == true && !empty? && !@ship.sunk?
```

Comment:
Good catch on the refactoring of render, we might need to revisit this again but overall were good for now.